### ISSUE TITLE
All Noctua curators should have a group associated in the YAML file

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-site#453]

**Comment by pgaudet** [comment]
Note: default group is the first in the list.

**Comment by vanaukenk** [comment]
Added to agenda for 2017-11-14 annotation call:

http://wiki.geneontology.org/index.php/Annotation_Conf._Call_2017-11-14

**Comment by kltm** [comment]
To clarify this a bit more, as policy moving forward, all curators need to have group information (at least one) specified in their `users.yaml`. This group also needs to be specified exactly the same way in the `groups.yaml`. As it stands now, the curator group information can be gappy.

Ideally, there is a contact person for each group that can help triage their own data in these files. For future use, it may be good to add the relevant metadata in the `comments` of the `users.yaml` and/or create a group for these people in GH.

**Comment by kltm** [comment]
As well, as an early workaround before we settled on ORCIDs, we allowed people to use either a URI or to fall back onto their GOC xrefs. I believe we should stop this practice and require all curators to have an identifying URI (very very preferably an ORCID). Any xref they have must be kept around so we can do a one-time forward mapping.

**Comment by kltm** [comment]
Also want to tag on @cmungall for that last bit.

**Comment by cmungall** [comment]
groups actually used in models:

```
http://dictybase.org
http://geneontology.org
http://informatics.jax.org
http://lbl.gov
http://purl.obolibrary.org/go/groups/BHF-UCL
http://purl.obolibrary.org/go/groups/IntAct
http://purl.obolibrary.org/go/groups/UniProt
http://purl.obolibrary.org/go/groups/WB
http://www.ucl.ac.uk/functional-gene-annotation/cardiovascular
http://www.ucl.ac.uk/functional-gene-annotation/neurological/projects/tabs/aruk-ucl
http://www.wormbase.org
http://www.yeastgenome.org
http://zfin.org
https://www.nextprot.org
https://www.ucl.ac.uk/functional-gene-annotation/neurological/projects/tabs/syngo-ucl
```

**Comment by kltm** [comment]
@cmungall Closure would be that all past and current curators have correct metadata.
https://github.com/geneontology/go-site/issues/453#issuecomment-343007547
It may be that this can be confirmed and has been done.

**Comment by kltm** [comment]
@vanaukenk @pgaudet 
We are currently finishing up updating users and groups, patching up as best we can. This is the current list of users we are planning to drop Noctua edit permissions for:
```
REPAIR?: Revoke Jim Knowles noctua-go edit privileges.
REPAIR?: Revoke Anushya Muruganujan noctua-go edit privileges.
REPAIR?: Revoke Anna Melidoni noctua-go edit privileges.
REPAIR?: Revoke Mike Livstone noctua-go edit privileges.
REPAIR?: Revoke Dexter Pratt noctua-go edit privileges.
REPAIR?: Revoke Michael Rice noctua-go edit privileges.
REPAIR?: Revoke Diane Inglis noctua-go edit privileges.
REPAIR?: Revoke Sejal Patel noctua-go edit privileges.
REPAIR?: Revoke Isabelle Cusin noctua-go edit privileges.
REPAIR?: Revoke Amanda Mackie noctua-go edit privileges.
REPAIR?: Revoke Brenley McIntosh noctua-go edit privileges.
REPAIR?: Revoke Nicole Vasilevsky noctua-go edit privileges.
REPAIR?: Revoke Prudence Mutowo noctua-go edit privileges.
REPAIR?: Revoke Yasmin Alam-Faruque noctua-go edit privileges.
REPAIR?: Revoke Ingrid Keseler noctua-go edit privileges.
REPAIR?: Revoke Dmitry Sitnikov noctua-go edit privileges.
REPAIR?: Revoke Grace Stafford noctua-go edit privileges.
REPAIR?: Revoke Claudia Rato da Silva noctua-go edit privileges.
REPAIR?: Revoke Magdalen Lindeberg noctua-go edit privileges.
REPAIR?: Revoke Peter D'Eustachio noctua-go edit privileges.
REPAIR?: Revoke Shur-Jen Wang noctua-go edit privileges.
REPAIR?: Revoke Jodi Hirschman noctua-go edit privileges.
REPAIR?: Revoke Anne Estreicher noctua-go edit privileges.
REPAIR?: Revoke Alan Bridge noctua-go edit privileges.
REPAIR?: Revoke Andre Stutz noctua-go edit privileges.
REPAIR?: Revoke Duncan Legge noctua-go edit privileges.
REPAIR?: Revoke Guillaume Keller noctua-go edit privileges.
REPAIR?: Revoke Michel Schneider noctua-go edit privileges.
REPAIR?: Revoke Michele Magrane noctua-go edit privileges.
REPAIR?: Revoke Shyamala Sundaram noctua-go edit privileges.
REPAIR?: Revoke Ursula Hinz noctua-go edit privileges.
REPAIR?: Revoke Mary L. Schaeffer noctua-go edit privileges.
REPAIR?: Revoke Nevila Hyka-Nouspikel noctua-go edit privileges.
REPAIR?: Revoke Pablo Porras Millan noctua-go edit privileges.
REPAIR?: Revoke Lionel Breuza noctua-go edit privileges.
REPAIR?: Revoke Trudy Torto noctua-go edit privileges.
REPAIR?: Revoke Helen Parkinson noctua-go edit privileges.
REPAIR?: Revoke Peter Robinson noctua-go edit privileges.
REPAIR?: Revoke Anne Niknejad noctua-go edit privileges.
```
As far as I know, none of these people are currently building models. As the majority of these users are being dropped for the reason of not having an ORCID and re-adding them is trivial, I think we probably just want to go ahead with the automated repairs that we have ready to go.

**Comment by kltm** [comment]
@vanaukenk That would be great. Or you could just email them to me--either would be fine.
I was also wondering about what to do about the `organization` of people who are no longer active; e.g. the user for GOC:di, who had conflicting information.

**Comment by vanaukenk** [comment]
For GOC:di, is it possible to include two distinct organization lines in the users.yaml for that entry?

Are there other types of issues wrt populating organization?

**Comment by kltm** [comment]
@vanaukenk The `organization` field is single-valued, and I'm not sure if we'd want to change that to support this single case where the person is very much no longer active with either of the groups (from linkedin). I'm wondering if we should have a special value there for for "unaffiliated" people or remove it (which we could use to flag a user for removal from permissions).

**Comment by kltm** [comment]
@vanaukenk 
That's a possibility (previous orgs): it would be the analog for the (not currently used but present in the schema) date range in the groups.yaml.
As it stands now, the history is kept in the git repo. Something tells me that we don't want to try and turn this metadata file into linkedin-lite, but I can't quite put my finger on why. Arguments against could be: it would take some work to keep it all up to date and we don't actually use `organization` for anything. Honestly, it could be completely captured in groups.
My "counter" thinking out loud would be to deprecate `organization` and allow people to identify at whatever granularity they want in `groups`. If there is need, we could add a `previous-groups` field or something.

**Comment by vanaukenk** [comment]
@kltm 
I like the "counter" proposal.  We could deprecate organization, especially if it's not being used for anything, and instead have groups and previous-groups tags.  The previous-groups tag would perhaps give us a place to start looking if we ever needed to follow up on an issue.  This would require some periodic curation of the users.yaml file but probably not too much and maybe could be automated in some way?  This is not a high priority, though, IMO....

**Comment by kltm** [comment]
@vanaukenk 
I would agree. Assuming that @cmungall can't think of some other use of `organization` that I have forgotten, I would suggest making `organization` and `previous-groups` optional fields and worry about it all a little later.

**Comment by vanaukenk** [comment]
Thanks for the update @kltm 
I'll let curators know about this on the next annotation call.
